### DISCUSSION TITLE
Exclude routes from OpenAPI spec

### DISCUSSION THREAD

**Discussion by and3rson** [0b01001001/spectree#384] [Q&A]
By default, all routes that are defined in Starlette application are included in the OpenAPI schema. Is it possible to specifically exclude some of them?
EDIT: Seems like overriding `bypass` method can do this, but it would be nice if `@spectree.validate()` allowed to mark method for exclusion from the OpenAPI schema.

**Comment by kemingy** [comment]
Refer to https://github.com/0b01001001/spectree/blob/78edc00241006832f32a0e569e7a27844ca9aa38/spectree/config.py#L75

You can use `spec = SpecTree(mode='strict')` to indicate that only the decorated route will be included.
